doc/source/whatsnew/v0.23.0.txt

```diff
@@ -474,3 +474,4 @@ Other
 
 - Improved error message when attempting to use a Python keyword as an identifier in a ``numexpr`` backed query (:issue:`18221`)
 - :func:`Timestamp.replace` will now handle Daylight Savings transitions gracefully (:issue:`18319`)
+- Bug in :class:`WeekOfMonth` and :class:`LastWeekOfMonth` where default keyword arguments for constructor raised ``ValueError`` (:issue:`19142`)
```

Comment:
pls move this and the one above to COnversion, don't put things Other unless they obviously don't fit elsewhere. You *could* make a TimesSeries section if you want (and move appropriate things), but move this for now.

Reply:
OK.  For future reference, what heuristic should I be using that maps `WeekOfMonth.__init__ ValueError` to "conversion"?